src/aws/mod.rs

```diff
@@ -337,7 +338,6 @@ impl Body for MeasuredBody {
             Poll::Ready(Some(Ok(data))) => {
                 this.shared_bytes_sent
                     .fetch_add(data.len(), Ordering::Release);
-
```

Comment:
**nitpicking**: What you don't like my use of whitespace?!

Reply:
wasn't intentional, probably occurred while fixing the merge conflicts.